horreum-web/src/components/ConditionComponent.tsx

```diff
@@ -8,45 +8,48 @@ type ConditionComponentProps = {
     value: any
     onChange(value: any): void
     isTester: boolean
+    properties: {
+        [key: string]: any
+    }
 } & ConditionComponentDef
```

Comment:
We do not need to override `properties`, it is already defined from the openAPI spec for `ConditionComponentDef`

Reply:
@shivam-sharma7 if you remove this override, then it looks good to merge, thanks

Reply:
Yes, Gotcha!